README.md

```
@@ -1 +1,466 @@
+
+[![Notion](https://img.shields.io/badge/프로젝트_스토리-Notion-000000?style=flat-square&logo=notion&logoColor=white)](https://slime-shirt-140.notion.site/Bracket-2a202aff8b528063be0ff432c107e135)
+
+</div>
+
+---
+
+## 개요
+
+Bracket은 Canon 미러리스 카메라를 iPhone으로 제어하며, 미리 설정한 3가지 프리셋을 자동으로 순환 적용하는 iOS 앱입니다. 촬영한 사진은 Vision Framework를 통해 자동 그룹화되어 빠른 선별이 가능합니다.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)